.github/workflows/msal-node-e2e.yml

```diff
@@ -209,9 +209,13 @@ jobs:
         os: [windows-latest, macos-latest]
         sample:
           - 'ElectronTestApp'
+          - 'ElectronSystemBrowserTestApp'
 
     name: ${{ matrix.sample }} - ${{ matrix.os }}
 
+    env:
+        PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
```

Comment:
do we really need these env vars outside of the sample working directory?

Reply:
Passing the env variables on package installation was breaking the tests. I need to declare the env variables before the package installation, so I moved the location of the variable to be declared before all steps. Let me know if you have a better suggestion, but according to the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idenv), it should be fine.

Reply:
@derisen it works fine when moving the env variables to the `Install Test Tools` step. Thanks for the feedback!